test/analysis/typeComplexityAnalysis.test.ts

```diff
@@ -797,20 +799,6 @@ describe('Test getQueryTypeComplexity function', () => {
             });
         });
 
-        /**
-         * FIXME: handle lists of unknown size. change the expected result Once we figure out the implementation.
-         */
-        xtest('with lists of unknown size', () => {
```

Comment:
I'd save leave this in here, in case a future implementation changes how these are handled. Total complexity should be 11

Reply:
ok